app/src/sharedTest/java/org/oppia/android/app/profileprogress/ProfilePictureActivityTest.kt

```diff
@@ -126,9 +146,26 @@ class ProfilePictureActivityTest {
 
   @Test
   fun testProfilePictureActivity_userImageIsDisplayed() {
-    launch<ProfilePictureActivity>(createProfilePictureActivityIntent(internalProfileId)).use {
-      onView(withId(R.id.profile_picture_image_view)).check(matches(isDisplayed()))
-    }
+    activityTestRule.launchActivity(createProfilePictureActivityIntent(internalProfileId))
+    testCoroutineDispatchers.runCurrent()
+    onView(withId(R.id.profile_picture_image_view)).check(matches(isDisplayed()))
+  }
+
+  @Test
+  fun testProfilePictureActivity_hasCorrectTitle() {
```

Comment:
```suggestion
  fun testProfilePictureActivity_toolbarHasCorrectTitle() {
```

(Otherwise it's ambiguous since the activity also has a title that's separately defined).

Reply:
Okay I will make these changes